### DISCUSSION TITLE
BinnerScalar

### DISCUSSION THREAD

**Discussion by Kendidi** [vaexio/vaex#1417] [Q&A]
Hi,

Can someone please kindly explain how **BinnerScalar** works?
When I plotted a heapmap with shape=256, I see the BinnerScaler kept being called.  I wonder what was it doing?  Thanks a lot in advance!

**Comment by maartenbreddels** [comment]
Hi, I was wondering where you see this, since it's in the C++ layer?
BinnerScalar maps data from [min, max] to bin number, which is [0, N+3], where the first two and last bins are resolved (nan/missing, out of bound, <min, bin0, .... binN, out of bound, >= max). Does that help you?
